screenpy/actions/log.py

```diff
@@ -37,8 +35,8 @@ def perform_as(self, the_actor: Actor) -> None:
         if isinstance(self.question, Answerable):
             self.question.answered_by(the_actor)
         else:
-            # must be a value instead of a question!
-            aside(f"the value is: {self.question}")
+            # must be a value instead of a Question!
+            aside(f"the value is: {represent_prop(self.question)}")
```

Comment:
Should the represent_prop be in the questions?

Reply:
Should it not be? I don't see a reason why it can't be, did you not intend for it to be used here?

Reply:
Oh i see what you're saying—just to clarify, in this specific case, a value was passed in to `See` instead of a Question. Something like:
```python
the_actor.should(
    See.the(noted("todo item saved message"), ReadsExactly("ToDo Item Saved!"))
)
```

Reply:
Ahh.. good point.  I was concerned that because it's here folks won't use it in their questions.